drivers/rn2xx3/rn2xx3.c

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_rn2xx3
+ * @{
+ *
+ * @file
+ * @brief       Driver implementation for the RN2483/RN2903 devices
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <string.h>
+#include <errno.h>
+
+#include "assert.h"
+#include "xtimer.h"
+#include "rn2xx3_params.h"
+#include "rn2xx3.h"
+#include "rn2xx3_internal.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#if ENABLE_DEBUG
+/* we are interested by the debug message in the sleep timer callback */
+#define ISR_STACKSIZE               (THREAD_STACKSIZE_DEFAULT)
```

Comment:
Setting this here doesn't change the ISR stacksize

Reply:
what's the best place for this then ?

Reply:
No good place for this. But here it is useless. :)
Maybe a warning comment before the debug.h include is better?

Reply:
added a comment explaining how to set it via CFLAGS (untested). So not sure if it works or not.

Reply:
I tested this morning and modifying the `ISR_STACKSIZE` has no effect because the debug message is displayed from the idle thread. Modifying `THREAD_STACKSIZE_IDLE` using CFLAGS works. I updated the comment accordingly.